src/wp-includes/block-editor.php

```diff
@@ -211,10 +211,19 @@ function get_default_block_editor_settings() {
 		);
 	}
 
+	$allowed_mime_types = get_allowed_mime_types();
+
+	// Iterate through mime types and check if they are supported by the image editor.
```

Comment:
```suggestion
	// Iterate through mime types and remove those unsupported by the image editor.
```

A little nitpicky, but imho that one is more into the point.